packages/components/progress-bar/src/progress-bar.ts

```diff
@@ -0,0 +1,98 @@
+import { localized, msg } from '@lit/localize';
+import { type ScopedElementsMap, ScopedElementsMixin } from '@open-wc/scoped-elements/lit-element.js';
+import { Icon } from '@sl-design-system/icon';
+import { type CSSResultGroup, LitElement, type TemplateResult, html, nothing } from 'lit';
+import { property } from 'lit/decorators.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import styles from './progress-bar.scss.js';
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'sl-progress-bar': ProgressBar;
+  }
+}
+
+export type ProgressState = 'active' | 'success' | 'warning' | 'error';
```

Comment:
Is this a state or a variant? 
I feel a state is something that changes based on a user action within the component. This is an input, a choice by the designer/developer.

Reply:
I was thinking about `status` there instead of `state`, but due to discussion with Danielle and to make it the same as in Figma I've changed it to `state`. @DanielleRameau what do you think about renaming it to `variant`?

We are also using state to inform a user that sth has changed (in aria-live) so maybe it would be better to leave `state` here?

Reply:
I like `variant`, but just because it would then match with other components that have `variant` and `success`, `warning` values. Not sure if that is reason enough. But i also agree with @Diaan .

Reply:
I've changed it to `variant` :)